File: PHP/src/Jakopec/PartijaDvaPara.php
<?php


namespace Jakopec;

use Doctrine\ORM\Mapping\Entity;

/**
 * @Entity
 */
class PartijaDvaPara extends Partija
{
    public function __toString(): string
    {
        $rezultat = $this->getRezultat();
        return 'Partija DVA PARA, igra gotova: ' . ($this->isIgraGotova() ? 'DA': 'NE') . ', ' . $this->getIgraci()[0] . ' i ' .
            $this->getIgraci()[1] . ': ' . $rezultat->getPrvi() .
            ' | ' . $this->getIgraci()[2] . ' i ' . $this->getIgraci()[3] . ': ' . $rezultat->getDrugi();
    }
}

File: Java/src/main/java/jakopec/model/PartijaDvaPara.java
package jakopec.model;


import jakarta.persistence.DiscriminatorValue;
import jakarta.persistence.Entity;
import jakarta.persistence.MappedSuperclass;
import jakarta.persistence.Table;

@Entity
@DiscriminatorValue("dvaPara")
public class PartijaDvaPara extends Partija {

    @Override
    public String toString() {
        Rezulat rezulat = getRezultat();
        return "Partija DVA PARA, igra gotova: " + (isIgraGotova() ? "DA" : "NE") + ", " +
                getIgraci().get(0) + " i " + getIgraci().get(1) + ": " + rezulat.getPrvi() + " | "+
                getIgraci().get(2) + " i " + getIgraci().get(3)  + ": " + rezulat.getDrugi();
    }


}
